src/tests/unit/mock_data/mock_blueprints/two_contained_deep_attributes.blueprint.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "two_contained_deep_attributes",
```

Comment:
What does "deep" mean?

Reply:
Naming improvements will come in future PR. I agree that it's not clear